app/controllers/customer_items_controller.rb

```diff
@@ -0,0 +1,8 @@
+class CustomerItemsController < ApplicationController
```

Comment:
Great controller organization! This is the most RESTful option - using a separate Joins controller with its own `create` action